Source/Krypton Toolkit/Krypton.Toolkit.Suite.Extended.Controls/Controls Toolkit/KryptonKnobControlVersion2.cs

```diff
@@ -1074,6 +1074,7 @@ private Point GetKnobPosition(int l)
         /// <summary>
         /// return 2 points of a line starting from the center of the knob to the periphery
         /// </summary>
+        /// <param name="Gr"></param>
         /// <param name="l"></param>
         /// <returns></returns>
         private Point[] GetKnobLine(Graphics Gr, int l)
```

Comment:
Parameters should be lowerCamelCase start

Reply:
Renamed to `g`